src/runtime/server/api/search.ts

```diff
@@ -0,0 +1,20 @@
+import { defineEventHandler } from 'h3'
+import { splitPageIntoSections } from '../search'
+import { useRuntimeConfig } from '#imports'
+import { serverQueryContent } from '#content/server'
+
+export default defineEventHandler(async (event) => {
+  const runtimeConfig = useRuntimeConfig()
+  const { ignoredTags } = runtimeConfig.public.content.search
+
+  const files = await serverQueryContent(event).find()
```

Comment:
It is safe if we filter `partial` and `drafted` contents from search by default.
As an advanced option, we can also provide config in module option to control behavior of this search by providing query

Reply:
Yes, I will add `partial` and `draft` by default and will see later if it's useful to use an option for this behavior.

Reply:
I added some options.